client/src/paths/editor/DocEditorRemixMode.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import axios from "axios";
+import { PanelPair } from "../../widgets/PanelPair";
+import { Remixes } from "../../drawerTabs/Remixes";
+import { RemixSources } from "../../drawerTabs/RemixSources";
+import { processRemixes } from "../../utils/processRemixes";
+import { useLoaderData, useNavigate, useOutletContext } from "react-router";
+import { ActivityRemixItem } from "../../types";
+import {
+  Box,
+  Center,
+  Heading,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+} from "@chakra-ui/react";
+import { EditorContext } from "./EditorHeader";
+
+export async function loader({ params }: { params: any }) {
+  const { data } = await axios.get(
+    `/api/remix/getRemixSources/${params.contentId}`,
+  );
+  const sources = processRemixes(data.remixSources);
+
+  const { data: data2 } = await axios.get(
+    `/api/remix/getRemixes/${params.contentId}`,
+  );
+  const remixes = processRemixes(data2.remixes);
+
+  return { remixes, sources };
+}
+
+/**
+ * This page allows you to view your remix sources and remixes.
+ * Context: `documentEditor`
+ */
+export function DocEditorRemixMode() {
+  const { remixes, sources } = useLoaderData() as {
+    remixes: ActivityRemixItem[];
+    sources: ActivityRemixItem[];
+  };
+
+  const { contentName } = useOutletContext<EditorContext>();
+
+  const haveChangedSource = sources.some(
+    (source) => source.originContent.changed,
+  );
+  const haveChangedRemix = remixes.some((remix) => remix.remixContent.changed);
+
+  const navigate = useNavigate();
+
+  const sourcesPanel = (
+    <Box margin="1rem">
+      <Center>
+        <Heading size="md">This document is remixed from:</Heading>
+      </Center>
+      <RemixSources
```

Comment:
It looks like `<RemixSources>` was not changed in this PR. However, now that data that it receives is not consistent. When viewing this panel from a document with a changed remix source, I see a notification dot on the remix button, and I get the message on this panel that the dot indcates a remix source has changed. However, it seems like `contributorHistory.originContent.changed` is not set, as there isn't the entry at the right with the notification dot and the link to the activity comparison screen.

Reply:
Yes, looked fixed!